curriculum/challenges/english/01-responsive-web-design/css-grid-magazine/part-011.md

```diff
@@ -0,0 +1,125 @@
+---
+id: 61439dfc811e12666b04be6f
+title: Part 11
+challengeType: 0
+dashedName: part-11
+---
+
+# --description--
+
+Add a third `p` element at the end of your `.text` element, and give it the following text:
+
+```
+It wasn't as dramatic as Doc's revelation in Back to the Future. It
+just occurred to me while I was going for a run. The revelation: the
+entire curriculum should be a series of projects. Instead of
+individual coding challenges, we'll just have projects, each with
+their own seamless series of tests. Each test gives you just enough
+information to figure out how to get it to pass. (And you can view
+hints if that isn't enough.)
```

Comment:
```suggestion
It wasn't as dramatic as Doc's revelation in Back to the Future. It just occurred to me while I was going for a run. The revelation: the entire curriculum should be a series of projects. Instead of individual coding challenges, we'll just have projects, each with their own seamless series of tests. Each test gives you just enough information to figure out how to get it to pass. (And you can view hints if that isn't enough.)
```